plugins/inlinetoolbar/plugin.js

```diff
@@ -105,6 +104,40 @@
 				this.parts.close.remove();
 			};
 
+			/**
+			 * @inheritdoc CKEDITOR.ui.balloonPanel#show
+			 * @member CKEDITOR.ui.inlineToolbarView
+			 */
+			CKEDITOR.ui.inlineToolbarView.prototype.show = function() {
+				if ( this.rect.visible ) {
+					return;
+				}
+				var that = this,
+					editable = this.editor.editable();
+				this._detachListeners();
+
+				this._listeners.push( this.editor.on( 'resize', function() {
+					that.attach( that._element, {
+						focusElement: false
+					} );
+				} ) );
+				this._listeners.push( editable.attachListener( editable.getDocument(), 'scroll', function() {
+					that.attach( that._element, {
+						focusElement: false
+					} );
+				} ) );
+				CKEDITOR.ui.balloonPanel.prototype.show.call( this );
+			};
+
+			/**
+			 * @inheritdoc CKEDITOR.ui.balloonPanel#hide
```

Comment:
I might be wrong here but AFAIR if you don't provide this block it will inherit the doc anyway.